### PR TITLE
new views

### PR DESCRIPTION
**Pull request by gerouvi**
Team invite view

### CODE REVIEW

File: src/components/OrganizationSaas/Dashboard/Create.tsx
Comment (selankon) [inline src/components/OrganizationSaas/Dashboard/Create.tsx:62]: Should use `ListItem` instead of `li`? https://v2.chakra-ui.com/docs/components/list/usage#unordered-list

File: src/components/OrganizationSaas/Dashboard/Layout.tsx
Comment (selankon) [inline src/components/OrganizationSaas/Dashboard/Layout.tsx:39]: To add translation to this array  you will need to use the `useTranslation` hook, so the array should be declared inside a component.

Also, it should to be in lowecase (`const cards`). Another good practice is to define the inner object type (to ensure that all objects have the same attributes), something like:

```ts
type CardProps = {
popular: boolean
title: string
// etc
}
const cards: CardProps[] = [
//...
]
```

File: src/components/OrganizationSaas/Dashboard/Votings.tsx
Comment (selankon) [inline src/components/OrganizationSaas/Dashboard/Votings.tsx:43]: You could style the Pagination component itself maybe?? I don't think that is a good practice to do this style inheritance when we can just pass props to the Pagination component, but maybe @elboletaire have another opinion

File: src/components/OrganizationSaas/Dashboard/Create.tsx
Comment (selankon) [inline src/components/OrganizationSaas/Dashboard/Create.tsx:156]: Mailto and this two urls should point to the correct place better

File: src/components/OrganizationSaas/Dashboard/Layout.tsx
Comment (selankon) [inline src/components/OrganizationSaas/Dashboard/Layout.tsx:302]: Is the contact us route already implemented? If is, navigate to it

File: src/themes/saas/components/Account/Create.tsx
Comment (selankon) [inline src/themes/saas/components/Account/Create.tsx:1]: not sure what is this component and why is here, does this do the same as signup? Or is another workflow?

Comment (selankon) [reply src/themes/saas/components/Account/Create.tsx:1]: And is correct to put this inside of themes folder?

File: src/themes/saas/components/modal.ts
Comment (selankon) [inline src/themes/saas/components/modal.ts:81]: In general, I don't think that do this is a good practice (styling the child using this kinds of css selectors). Maybe @elboletaire can suggest a better way to do this? Or we could just ignore it and go forward? 

Maybe creating a component and style the inner children it via props is a better approach?

**Review by selankon — CHANGES_REQUESTED** [review]
I found that lots of `Trans` component have not the `i18nKey` prop, is this deliberated?

File: src/components/OrganizationSaas/Dashboard/Layout.tsx
Comment (gerouvi) [reply src/components/OrganizationSaas/Dashboard/Layout.tsx:39]: Okay, Aulet told me that he would review the translations later. For now, I've left it so that it looks good visually

File: src/components/OrganizationSaas/Dashboard/Votings.tsx
Comment (elboletaire) [reply src/components/OrganizationSaas/Dashboard/Votings.tsx:43]: Yeah, please, style components with their corresponding theme keys. Stop doing CSS in components with inheritance.

File: src/components/OrganizationSaas/Dashboard/Create.tsx
Comment (elboletaire) [reply src/components/OrganizationSaas/Dashboard/Create.tsx:156]: Also, if it's an external link, always use `isExternal`.

File: src/themes/saas/components/modal.ts
Comment (gerouvi) [reply src/themes/saas/components/modal.ts:81]: if I'm not mistaken, when creating a variant (of the modal in this case), we have to do it this way, since the parts of a modal (header, body, etc.) only extend so far, and to style the internal components, we need to access them in this way.

File: src/components/OrganizationSaas/Dashboard/Layout.tsx
Comment (gerouvi) [reply src/components/OrganizationSaas/Dashboard/Layout.tsx:302]: It's not implemented yet, but I have set as={ReactRouterLink} so that it's ready. Thanks.

File: src/themes/saas/components/Account/Create.tsx
Comment (elboletaire) [reply src/themes/saas/components/Account/Create.tsx:1]: It is correct if the view requires a complete different layout for one theme compared to the other. But this specific view might need to be loaded when BACKEND_URL is defined, rather than given the theme saas.

Comment (gerouvi) [reply src/themes/saas/components/Account/Create.tsx:1]: It's another component https://github.com/vocdoni/interoperability/issues/201

File: src/themes/saas/components/modal.ts
Comment (elboletaire) [reply src/themes/saas/components/modal.ts:81]: Ideally, we should be using chakra-ui theming system (with multipart and normal). 

So rather than using `& > p` ideally we should define a section for that theme, and style it with the multi-part system, by name.

This will also help us in the future improve the theme and find issues, that otherwise would be hard to find due to the excess of CSS traversing.

**Review by elboletaire — CHANGES_REQUESTED** [review]

File: src/components/OrganizationSaas/Dashboard/Layout.tsx
Comment (selankon) [reply src/components/OrganizationSaas/Dashboard/Layout.tsx]: But you can just do like this

```
t('key.for.translation', { defaultValue: 'String you want to show' })
```

Then its easy to create the translations

File: src/themes/saas/components/modal.ts
Comment (selankon) [reply src/themes/saas/components/modal.ts:81]: To do not block this PR anymore I preffer to accept it and from now try to do this as you said, using the multipart component.

I am going to mark this as resolved

File: src/components/OrganizationSaas/Dashboard/Invite.tsx
Comment (selankon) [inline src/components/OrganizationSaas/Dashboard/Invite.tsx:50]: Label should be inside a translation also

File: src/elements/LayoutAuth.tsx
Comment (selankon) [inline src/elements/LayoutAuth.tsx:91]: missing `isExternal`

**Review by selankon — CHANGES_REQUESTED** [review]

File: src/themes/saas/components/Account/Create.tsx
Comment (selankon) [reply src/themes/saas/components/Account/Create.tsx:1]: I will do this

File: src/components/OrganizationSaas/Dashboard/Layout.tsx
Comment (selankon) [reply src/components/OrganizationSaas/Dashboard/Layout.tsx]: 18fe74e

File: src/components/OrganizationSaas/Dashboard/Invite.tsx
Comment (selankon) [reply src/components/OrganizationSaas/Dashboard/Invite.tsx]: 18fe74e

File: src/elements/LayoutAuth.tsx
Comment (selankon) [reply src/elements/LayoutAuth.tsx]: 18fe74e

File: src/themes/saas/components/Account/Create.tsx
Comment (selankon) [reply src/themes/saas/components/Account/Create.tsx]: 18fe74e

File: src/components/AccountSaas/Create.tsx
Comment (jpaulet) [inline src/components/AccountSaas/Create.tsx:1]: IMO this view name should be more verbose, like "CreateOrg" or similar, in order to help finding it when needed.

File: src/components/OrganizationSaas/Dashboard/Create.tsx
Comment (jpaulet) [inline src/components/OrganizationSaas/Dashboard/Create.tsx:139]: Typo: "costs"

File: src/components/AccountSaas/Create.tsx
Comment (selankon) [reply src/components/AccountSaas/Create.tsx:1]: In the actual scaffold this component has exactly this name:

`src/components/Account/Create.tsx`

I won't change it to follow same structure.

**Review by jpaulet — APPROVED** [review]

**Review by selankon — APPROVED** [review]

**Pull request merged**
